NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml

```diff
@@ -45,95 +45,102 @@
+                android:id="@+id/information_consent_lookup_text_textview"
+                style="@style/PrimaryTextRegular"/>
+			<TextView
+				android:text="Notification header"
 				android:layout_width="wrap_content"
 				android:layout_height="wrap_content"
-                android:layout_marginLeft="24dp"
-                android:layout_marginRight="24dp"
-                android:layout_below="@id/information_consent_body_one_bullet_linearLayout"
-				android:id="@+id/information_consent_body_two_bullet_linearLayout">
-                <ImageView 
-                    style="@style/Checkmark"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:src="@drawable/check_shield"/>
-                <TextView
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="Body two"
-                    android:id="@+id/information_consent_body_two_textView"
-                    style="@style/CheckmarkText"/>
-            </LinearLayout>
+				android:paddingLeft="24dp"
+				android:paddingRight="24dp"
+				android:paddingTop="24dp"
+                android:textSize="16sp" 
+                android:layout_below="@id/information_consent_lookup_text_textview"
+                android:id="@+id/information_consent_notification_header_textview"
+                style="@style/PrimaryTextBold"/>
             <TextView
-				android:text="Content"
+				android:text="Notification text"
 				android:layout_width="wrap_content"
 				android:layout_height="wrap_content"
-                android:layout_margin="24dp"
+                android:paddingLeft="24dp"
+				android:paddingRight="24dp"
                 android:textSize="16sp" 
-                android:layout_below="@id/information_consent_body_two_bullet_linearLayout"
-                android:id="@+id/information_consent_content_two_textView"
+                android:layout_below="@id/information_consent_notification_header_textview"
+                android:id="@+id/information_consent_notification_text_textview"
                 style="@style/PrimaryTextRegular"/>
-        </RelativeLayout>
+			<TextView
+				android:text="Be aware text"
+				android:layout_width="wrap_content"
+				android:layout_height="wrap_content"
+				android:paddingTop="48dp"
+                android:paddingLeft="24dp"
+				android:paddingRight="24dp"
+                android:textSize="16sp" 
+                android:layout_below="@id/information_consent_notification_text_textview"
+                android:id="@+id/information_consent_beAware_text_textview"
+                style="@style/PrimaryTextBold"/>
+            <TextView
+				android:text="Notification text"
+				android:layout_width="wrap_content"
+				android:layout_height="wrap_content"
+				android:paddingTop="24dp"
+                android:paddingLeft="24dp"
```

Comment:
the same as above

Reply:
Hmmm.. weird. That must be Github doing something.
![image](https://user-images.githubusercontent.com/60694331/99500989-6b216900-297b-11eb-9483-39aa32c33be2.png)